src/components/Agreements/AgreementForm/Sections/AgreementFormOrganizations.js

```diff
@@ -90,24 +80,11 @@ class AgreementFormOrganizations extends React.Component {
           { renderedOrgs.map((org, index) => (
             <Row key={index}>
               <Col xs={8}>
-                <FormattedMessage id="ui-agreements.organizations.selectOrg">
-                  {placeholder => (
-                    <Field
-                      name={`orgs[${index}].org`}
-                      component={Selection}
-                      dataOptions={this.state.orgs}
-                      formatter={(props) => {
-                        return <OptionSegment {...props}>{props.option.label}</OptionSegment>;
-                      }}
-                      onFilter={(searchString, orgs) => {
-                        return orgs.filter(o => {
-                          return o.label.toLowerCase().includes(searchString.toLowerCase());
-                        });
-                      }}
-                      placeholder={placeholder}
-                    />
-                  )}
-                </FormattedMessage>
+                <Field
+                  component={OrganizationSelection}
+                  name={`orgs[${index}].org`}
+                  stripes={this.props.stripes}
```

Comment:
I've been preferring a pattern of using `withStripes()` inside `OrganizationSelection` instead of prop-drilling the `stripes` god object down into children.